src/main/scala/com/gu/datalakealerts/Features.scala

```diff
@@ -138,4 +138,61 @@ object Features {
       }
     }
   }
+  case object OlgilBanner extends Feature {
+    override val id = "olgil_banner"
+
+    override def monitoringQueryResult(resultSet: ResultSet, minimumImpressionsThreshold: Int): MonitoringQueryResult = {
+      val impressionCountsByAppVersion = ImpressionCounts.getImpressionCounts(resultSet)
+      val totalImpressions = impressionCountsByAppVersion.map(_.impressions).sum
+      val resultIsAcceptable = totalImpressions > minimumImpressionsThreshold
+      MonitoringQueryResult(resultIsAcceptable, AlertInformation.describeResults(totalImpressions, minimumImpressionsThreshold))
+    }
+
+    override def monitoringQuery(platform: Platform): MonitoringQuery = {
+      platform match {
+        case iOS =>
+          MonitoringQuery(s"""
+            |select browser_version, count (distinct page_view_id) as banner_impressions
+            |from clean.pageview
+            |cross join unnest (ab_tests) x (ab)
+            |where received_date = date '$yesterday'
+            |and path not like '%.mp3%'
+            |and device_type like '%IOS%'
+            |and ab.name like '%banner%'
+            |and ab.completed = False
+            |group by 1
+          """.stripMargin, 33280)
+        case _ => throw new RuntimeException("Only iOS platform is supported.")
+
+      }
+    }
+  }
+
+  case object BrazeBanner extends Feature {
+    override val id = "braze_banner"
+
+    override def monitoringQueryResult(resultSet: ResultSet, minimumImpressionsThreshold: Int): MonitoringQueryResult = {
+      val impressionCountsByAppVersion = ImpressionCounts.getImpressionCounts(resultSet)
```

Comment:
Although this isn't a new problem introduced by this PR, this code is now duplicated several times. We should think about how this code can be shared between features (probably best for that to be addressed in a separate PR though)